configs/config-combinations/README.md

```diff
@@ -0,0 +1,21 @@
+This directory contains YAML files which are used to track dependencies for 
```

Comment:
change the name of the folder from config-combinations to something else

Reply:
Done